Scripts/Runtime/Jobs/Deferred/IJobDeferredNativeArrayForBatch.cs

```diff
@@ -0,0 +1,140 @@
+using Anvil.Unity.DOTS.Collections;
+using System;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using Unity.Burst;
+using Unity.Collections;
+using Unity.Collections.LowLevel.Unsafe;
+using Unity.Jobs;
+using Unity.Jobs.LowLevel.Unsafe;
+using UnityEngine.Scripting;
+
+namespace Anvil.Unity.DOTS.Jobs
```

Comment:
{Insert copy of all comments from `IJobDeferredNativeArrayFor.cs` here}

Reply:
Done